### ISSUE TITLE
libfpx currently fails to install

### ISSUE THREAD

**Issue by dbermond** [rmarquis/pacaur#781]
###### Version
```
$ pacaur -v
pacaur 4.7.10
```

###### Description
[libfpx](https://aur.archlinux.org/packages/libfpx/) currently builds fine with `makepkg -o && makepkg -es` but fails to _install_ with pacaur.

It builds fine with pacaur, but fails at the install step with:
```
$ pacaur -S libfpx
...
:: Installing libfpx package(s)...
:: libfpx package(s) failed to install.
:: ensure package version does not mismatch between .SRCINFO and PKGBUILD
:: ensure package name has a VCS suffix if this is a devel package
```

Upstream libfpx does not provide a download archive of previous versions. It provides only the latest version. Being such, the PKGBUILD has a pkgver that is dynamically adjusted at build time if there is a new upstream release in order to allow the user to build and install the package even if there is a new upstream release.

Otherwise, package would fail to build when a new upstream release is made and the PKGBUILD is not yet updated, and not always the package maintainer can handle the version upgrade in the proper time.

I suppose that this issue occurs only in the case that there is a new upstream release and the package is not yet updated.

**Comment by rmarquis** [comment]
> I suppose that this issue occurs only in the case that there is a new upstream release and the package is not yet updated.

No, the issue is that the AUR see the package version as 1.3.1.9-1, as displayed on the [AUR page](https://aur.archlinux.org/packages/libfpx/). This is also the version provided by the [RPC interface](https://aur.archlinux.org/rpc/?v=5&type=info&arg[]=libfpx) that pacaur solely relies on by design for a bunch of different reasons, and which is obsolete in this case.

This version number is provided through the .SRCINFO file, which is normally updated at the same time the `pkgver` in the PKGBUILD is updated.

Though this is a edge case, I don't think there is a reason not to use a proper `pkgver`, even if upstream doesn't provide old archives (though that is a _real_ shame).

**Comment by dbermond** [comment]
Thank you for the reply.

> Though this is a edge case, I don't think there is a reason not to use a proper pkgver, even if upstream doesn't provide old archives (though that is a real shame).

In a package with a similar situation ([imagemagick-full](https://aur.archlinux.org/packages/imagemagick-full/)), people started to report broken package in the situation where there was a new upstream release and I could not yet update the package. This is an uncomfortable situation, because packages should continue to build fine even when being out-of-date, otherwise package will be broken every time that there is a new upstream release (and imagemagick releases new versions very frequently!). So I changed PKGBUILD to this dynamic pkgver schema and followed libfpx with the same logic.

I have updated libfpx to version 1.3.1.10, which is the version currently available at upstream, and pacaur installs it fine. But it would be good to allow installation with pacaur when the package is out-of-date too.

Glad to see that you added this to the '4.8.x new features' milestone. I hope this can be addressed in the future.
